src/Extractor/BaseExtractor.php

```diff
@@ -99,6 +103,7 @@ protected function createManifestGenerator(): ManifestGenerator
         return new DefaultManifestGenerator(
             $this->getMetadataProvider(),
             new DefaultManifestSerializer(),
+            $this->extractorClass,
```

Comment:
tohle si mohl rovnou vzít z $this->parameters['extractor_class'] a ušetřil by sis přepisování class v jednotlivých extractorech

Reply:
True, opraveno